graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -559,6 +567,7 @@ def _write_df(
         if out_format == "parquet":
             # Write to parquet
             input_df = self._replace_special_chars_in_cols(input_df)
+            # TODO: Remove overwrite mode
```

Comment:
Any reason we would like to remove overwrite? I would say overwrite would be most recommended.

Reply:
Yes, one thing we've observed (myself and with end users) is that they often run the same job more than once, keeping the same output-prefix (for example an error happens during a run, they fix part of the config and run again).

This results in some input files being duplicated under the same path, e.g. raw_id_mappings will have files for both names. When partition tries to read that it gets 2x the number of rows, which leads to errors. 

It's better if we enforce users to either clean output directories or select new ones to avoid this.